puppetlabs_spec_helper.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.email = ["modules-dept@puppetlabs.com"]
   s.extra_rdoc_files = [
     "LICENSE",
-    "README.markdown"
+    "README.md"
```

Comment:
Thanks for updating this, but please do not commit changes unrelated to your feature, to avoid maintenance issues later.